### PR TITLE
Enhanced Privacy Page in new VigyBag

### PR DESCRIPTION
**Pull request by SadafKausar2025**
## Fixes Issue #1472

 #1472

## Screenshots

https://github.com/codervivek5/VigyBag/assets/137484958/ef01f36e-bf75-402c-83d0-9e0110595786

## Summary by CodeRabbit

- **New Features**
  - Added "Privacy Policy" and "Return and Cancellation" links to the Footer.
  - Enhanced the Privacy Policy page with detailed sections on data collection, cookies, data sharing, security, user consent, and more.
  
- **Improvements**
  - Updated Privacy Policy page to load scripts dynamically for better performance.
  - Improved address display format in the Footer for readability.
  - Applied consistent styling rules to enhance visual presentation of the Privacy Policy page.

### CODE REVIEW

**Comment by SadafKausar2025** [comment]
hello @codervivek5 
please review this PR and merge it

File: src/components/Footer/Footer.jsx
Comment (codervivek5) [inline src/components/Footer/Footer.jsx:1]: why you replace all  ' '  to  " "

File: src/pages/Privacy-Policy/Privacy.jsx [hunk 1]
```diff
@@ -1,11 +1,322 @@
...
+          <div className="head">
+            <h1>VigyBag Privacy Policy</h1>
+          </div>
+          <section>
+            <p>
+              We value the trust you place in us and recognize the importance of
+              secure transactions and information privacy. This Privacy Policy
+              describes how VigyBag E-Commerce Pvt. Ltd. and its affiliates
+              (collectively “VigyBag, we, our, us”) collect, use, share, or
+              otherwise process your personal data through the VigyBag website{" "}
```
Comment (codervivek5) [inline src/pages/Privacy-Policy/Privacy.jsx]: website{" "}

what is it mean?

File: src/pages/Privacy-Policy/Privacy.jsx [hunk 2]
```diff
@@ -1,11 +1,322 @@
...
+            <p>
+              We may share personal data internally within VigyBag group
+              companies and third parties, including Credit Bureaus and business
+              partners, for purposes of providing products and services offered
+              by them. These entities and affiliates may share such information
+              with their affiliates, business partners, and other third parties
+              for the purpose of conducting the required checks, namely for the
+              purpose of credit underwriting, providing you their products and
+              services, and may market to you as a result of such sharing unless
+              you explicitly opt out.{" "}
```
Comment (codervivek5) [inline src/pages/Privacy-Policy/Privacy.jsx]: check this line for {" "}

File: src/pages/Privacy-Policy/Privacy.jsx [hunk 3]
```diff
@@ -1,11 +1,322 @@
...
+              We value the trust you place in us and recognize the importance of
+              secure transactions and information privacy. This Privacy Policy
+              describes how VigyBag E-Commerce Pvt. Ltd. and its affiliates
+              (collectively “VigyBag, we, our, us”) collect, use, share, or
+              otherwise process your personal data through the VigyBag website{" "}
+              <a className="a">https://www.vigybag.com/</a>, its mobile
+              application, and m-site (hereinafter referred to as the
+              “Platform”). While you can browse sections of the Platform without
+              sharing any information with us, please note that we do not offer
+              any product or service under this Platform outside <b>India</b>{" "}
```
Comment (codervivek5) [inline src/pages/Privacy-Policy/Privacy.jsx]: check this line for {" "}

File: src/pages/Privacy-Policy/Privacy.jsx [hunk 4]
```diff
@@ -1,11 +1,322 @@
...
+            <h2>Grievance Officer</h2>
+            <p>
+              In accordance with the Information Technology Act, 2000 and the
+              rules made thereunder, the name and contact details of the
+              Grievance Officer are provided below:
+              <p>
+                <b>Name:</b> Vivek Prajapati
+              </p>
+              <p>
+                <b>Email:</b> <a className="a">grievanceofficer@vigybag.com</a>{" "}
```
Comment (codervivek5) [inline src/pages/Privacy-Policy/Privacy.jsx]: check this line for {" "}

Comment (codervivek5) [reply src/pages/Privacy-Policy/Privacy.jsx]: why are you not make this for mail:
see below code 

<a className="a" href="mailto:grievanceofficer@vigybag.com">grievanceofficer@vigybag.com</a>

File: src/pages/Privacy-Policy/Privacy.jsx [hunk 5]
```diff
@@ -1,11 +1,322 @@
...
+              </p>
+            </p>
+          </section>
+          <section>
+            <h2>Queries</h2>
+            <p>
+              f you have a question, issue, complaint, or feedback in relation
+              to the processing of your personal data or this Privacy Policy,
+              you may reach out to our data protection officer at
+              <a className="a">grievanceofficer@vigybag.com.</a>{" "}
```
Comment (codervivek5) [inline src/pages/Privacy-Policy/Privacy.jsx]: fix here also 
<a className="a" href="mailto:grievanceofficer@vigybag.com">grievanceofficer@vigybag.com</a>

**Review by codervivek5 — CHANGES_REQUESTED** [review]
Take a look of these changes ...

**Review by codervivek5 — APPROVED** [review]

**Pull request merged**